uint16 itis
int32 latitude
int32 longitude
uint16 extent
